access-grant/pom.xml

```diff
@@ -98,9 +122,124 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <configuration>
-          <systemPropertyVariables />
-        </configuration>
+        <executions>
+          <execution>
+            <id>default-test</id>
+            <phase>none</phase>
+          </execution>
+          <!--<execution>
```

Comment:
This commented-out section can be removed

Reply:
Done in [d90f442](https://github.com/inrupt/solid-client-java/pull/483/commits/d90f442745c35dfe21870f5281b3accafe45c966)